Dev/Install/install.php5

```diff
@@ -152,7 +152,7 @@ if(isset($_REQUEST['db'])) {
 		$databaseConfig['type'] = $type;
 	}
 } else {
-	$type = $_REQUEST['db']['type'] = defined('SS_DATABASE_CLASS') ? SS_DATABASE_CLASS : 'MySQLDatabase';
+	$type = $_REQUEST['db']['type'] = defined('SS_DATABASE_CLASS') ? SS_DATABASE_CLASS : 'MySQLPDODatabase';
```

Comment:
This code is hard to read isn't it?

I wonder if we could automatically guess default installer based on dependency detection; It could be safer to fall back to mysqldatabase if the pdo_mysql driver isn't installed.